src/crimson/os/seastore/cache.cc

```diff
@@ -394,6 +394,35 @@ Cache::replay_delta(
   }
 }
 
+Cache::get_next_dirty_extents_ret Cache::get_next_dirty_extents(
+  journal_seq_t seq)
+{
+  std::vector<CachedExtentRef> ret;
+  for (auto i = dirty.begin(); i != dirty.end(); ++i) {
+    CachedExtentRef cand;
+    if (i->dirty_from < seq) {
```

Comment:
could you add a comment in the header file for
```c++
CachedExtent::list dirty;
```

to emphasis that the extents in `dirty` are ordered by `dirty_from` as new dirty extents are always appended to this list?